trulens_eval/examples/experimental/virtual_example.ipynb

```diff
@@ -0,0 +1,249 @@
+{
```

Comment:
<p>Can we change this setup to more typical setup for running from installed package?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/806/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Will have version outside of experimental examples for that.